tests/test_sklearn_stacking.py

```diff
@@ -270,6 +351,83 @@ def converter(scope, operator, container):
         got = sess.run(None, {'X': x})[0]
         self.assertEqual(got.shape[0], x.shape[0])
 
+    @unittest.skipIf(StackingClassifier is None,
+                     reason="new in 0.22")
+    @ignore_warnings(category=FutureWarning)
+    def test_concat_stacking_passthrough(self):
+
+        class CustomTransformer:
```

Comment:
Just curious, why not inheriting from Estimator, TransformerMixin instead of implementing a new parser?